ReactCommon/react/renderer/mapbuffer/MapBufferBuilder.cpp

```diff
@@ -35,7 +35,7 @@ MapBufferBuilder::MapBufferBuilder(uint16_t initialSize) {
 void MapBufferBuilder::ensureKeyValueSpace() {
   int32_t oldKeyValuesSize = keyValuesSize_;
   react_native_assert(
-      (keyValuesSize_ < std::numeric_limits<uint16_t>::max() / 2) &&
```

Comment:
I added these before, but they got changed.
Solving this inequality by multiplying times 2, we get:
`keyValuesSize_ < std::numeric_limits::max()`
which would be fine, and doesn't detect overflows.

Reply:
I think < is correct, if you look at the definition of react_native_assert, it asserts if the condition is false

Reply:
https://github.com/facebook/react-native/blob/3c880a800937c604ee0ba1c81187941f07e6ead1/ReactCommon/react/debug/react_native_assert.h#L55:9

Reply:
Good catch, will change this now